From c010cc07401b372cd51d6623cfe18cbc43848af6 Mon Sep 17 00:00:00 2001
From: Yaroslav Lavrinenko <y.lavrinenko@samsung.com>
Date: Tue, 11 Oct 2022 16:21:07 +0300
Subject: [PATCH 08/27] [axdimm] Mount sysfs path for axdimm test apps

Add sysfs mount commant to secndp manifest. This allows to handle latest
changes in axdimm library.

Resolve: AXDIMM-440
---
 application/axdimm_secndp_test/secndp.manifest.template | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/application/axdimm_secndp_test/secndp.manifest.template b/application/axdimm_secndp_test/secndp.manifest.template
index 8ffbe605..e8508b67 100644
--- a/application/axdimm_secndp_test/secndp.manifest.template
+++ b/application/axdimm_secndp_test/secndp.manifest.template
@@ -23,6 +23,7 @@ fs.mounts = [
   { uri = "file:{{ axdimm_driver }}/build/test", path = "{{ axdimm_driver }}/build/test" },
   { uri = "file:{{ axdimm_driver }}/build/ai", path = "{{ axdimm_driver }}/build/ai" },
   { uri = "file:{{ axdimm_driver }}/build/secure", path = "{{ axdimm_driver }}/build/secure" },
+  { uri = "file:/sys/bus/", path = "/sys/bus/", type="chroot" }, #unsafe but we can't mount single file from sysfs
   { type = "tmpfs", path = "/tmp" }
 ]
 
@@ -32,6 +33,10 @@ sgx.thread_num = 512
 sgx.debug = true
 sgx.insecure__enable_shared_range = "true"
 
+sgx.allowed_files = [
+  "file:/sys/bus/dax/drivers/device_dax/dax0.0/ranks/"
+]
+
 sgx.trusted_files = [
   "file:/dev/",
   "file:{{ gramine.libos }}",
-- 
2.34.1

